### ISSUE TITLE
EL in MetaQA

### ISSUE THREAD

**Issue by rhoi2021** [yuyuz/MetaQA#4]
Hello @yuyuz 
I greatly appreciate it if you answer my question.
Each question has a topic entity which shows the main focus of the question. I would like to know which Entity Linking technique is used to identify the topic/main entities of MetaQA datasets.
I am looking forward to hearing from you.

**Comment by yuyuz** [comment]
Hi @rhoi2021

Thanks for your interest in MetaQA! Since MetaQA is a synthetic dataset, we inherit the entities from the knowledge base in the Facebook MovieQA (a.k.a. WikiMovies) dataset, which is also included as a file "kb.txt" in MetaQA. For the textual questions in MetaQA, we mark the topic entity with a pair of brackets in the question. For example, "what films did [Michelle Trachtenberg] star in". Please let me know if this helps : )

**Comment by rhoi2021** [comment]
It means you manually interlinked the questions' mentions to entities of the Facebook MovieQA (a.k.a. WikiMovies).

**Comment by yuyuz** [comment]
Yes, the MetaQA questions are synthetically constructed (via textual templates), and replace the entity placeholder with the entities from the WikiMovies knowledge base.
